tensorflow/core/kernels/mkl_fused_batch_norm_op.cc

```diff
@@ -1330,8 +1706,10 @@ class MklFusedBatchNormGradOp : public OpKernel {
     dnn_shape_diff_src.SetMklTensor(false);
     AllocateOutputSetMklShape(context, kDiffSrcIndex, diff_src_tensor,
                               tf_shape_src, dnn_shape_diff_src);
-    for (size_t i = 0; i < (*diff_src_tensor)->shape().num_elements(); i++)
-      (*diff_src_tensor)->flat<T>().data()[i] = 0;
+    int num_elements = (*diff_src_tensor)->shape().num_elements();
+    auto diff_src_data = (*diff_src_tensor)->flat<T>().data();
+    for (size_t i = 0; i < num_elements; i++)
```

Comment:
I would use std::fill instead of a loop. Same everywhere.

Reply:
Sure, I will do. 
There are places in files which are not changed in this PR. Should I do the same change?

Reply:
Please do. :-)

Reply:
I have done code refactoring for multiple places in mkl_fused_batch_norm_op.cc. 

To avoid potential merge conflicts, I did not apply this code change recommendation to
source files not related to this PR. 

We will include this suggestion, along with others (such as changing signature of Execute()
with proper const or non-const argument declarations), to create a separate "code clean up"
PR. 

Thanks!